### ISSUE TITLE
Suomenkielen käännös

### ISSUE THREAD

**Issue by hvesalai** [jannecederberg/koodaustunti.fi#3]
Hei, vielä on paljon käännettävää ja tarkastettavaa. Olen ahkeroinut käännösten parissa jo muutaman illan useamman viikon ajan, joten kaikki kommentit ovat tervetulleita (liittykää code.org:n crowdin.net-projektiin).

En myöskään pahastuisi jos lisäisitte maininnan käännöstyöstäni tekijät-sivulle.

p.s. issueta muokattu

**Comment by jannecederberg** [comment]
Joo, totta, pitää tehdä joku erillinen Tekijät-sivu ja ehdottomasti mainita
sut siellä myös!

Katsoinkin kääntäjälistaa crowdin:ssä ja nimi jäi mieleen :)

**Comment by hvesalai** [comment]
Lisäsin ja korjailin taas tänään monta käännöstä. Aika vaikeaa tuo on tehdä selkokielellä ja pitää koko ajan mielessä pikkulapset. Onko teillä mitään havaintoa kuinka usein noi muutokset otetaan käyttöön?

**Comment by jannecederberg** [comment]
Ei oo mulla kyllä ainakaan käsitystä refresh-syklistä. Loin Tekijät-sivun (http://koodaustunti.fi/tekijat/), johon on nyt linkki sivun footerissa. Kommentteja tai korjausehdotuksia em. sivuun?

**Comment by hvesalai** [comment]
Näyttää hyvältä. Sain juuri loput kooditehtäviin liittyvistä käännöksistä tehtyä. Vielä olisi "slidet" (slides.yml) jäljellä, eli ne eri asiantuntijoiden (Bill Gates, yms) sanomat asiat. Ne pitää kaikki käydä läpi, koska osassa niistä on suomennoksena englanninkielistä tekstiä.

**Comment by jannecederberg** [comment]
Ok, cool

**Comment by jannecederberg** [comment]
Morjes Heikki, miten homma etenee?

**Comment by hvesalai** [comment]
Edelleen ne slidet (slides.yml) tekemättä. Pistin kahdelle muulle kääntäjälle viestiä, mutta kenelläkään ei ole ilmeisesti juuri nyt aikaa. Työtä on varmaan jäljellä yhteensä parin tunnin edestä.

Pistin myös code.org:iin kyselyn siitä julkistussyklistä, mutta ei vastausta.

**Comment by hvesalai** [comment]
Nyt on kaikki käännetty. Vielä pitää jonkun käydä äänestämässä +1:stä kaikkiin ja keksiä parempi käännös "of" sanalle (esim. "1 of 10"), ellei minun "/" kelpaa.

Kääntäjiä on myös tullut pari lisää, joten heidät voi lisätä tekijät-sivulle.

**Comment by jannecederberg** [comment]
Eli pitääks joka ikiselle käännetylle stringille olla ainakin yks tykkäys/+ ennenku menee läpi?

**Comment by hvesalai** [comment]
Näin mä käsittäisin ton. Siellä on se Vote-näkymä, jossa on melko nopeaa antaa kaikille +/- tai antaa kilpaileva ehdotus (klikkaa suomenkielistä tekstiä).

**Comment by jannecederberg** [comment]
Kävin ajelemassa "All strings" -näkymästä tolla selaimen konsolista käsin kaikki läpi: jQuery(".active_suggestion_plus_button").each(function() { jQuery(this).click(); });

Yks merkkijono tuli ilmisesti just lisää: [Glossaryyn](http://crowdin.net/translate/codeorg/47/enus-fi) "set color". Käänsin "aseta väri", mutta jonkun pitäs äänestää sitä.

Kuka noi approovaa? Code.org:n henkilökunta varmaan.

**Comment by hvesalai** [comment]
:)

**Comment by hvesalai** [comment]
Approovauksesta en tiedä. Pistin jo useampaa kanavaa pitkin kyselyn code.org:iin

**Comment by jannecederberg** [comment]
Jees, suljen tän, mutta ennen sitä, onko uusia ajatuksia tätä syksyä silmälläpitäen?

**Comment by hvesalai** [comment]
Paljon uutta käännettävää sinne on tullut.

**Comment by jannecederberg** [comment]
Joo, totta, onks sulla kiinnostusta kääntää/organisoida?
